editor/editor_inspector.cpp

```diff
@@ -2755,8 +2756,14 @@ void EditorInspector::update_tree() {
 					name = EditorNode::get_editor_data().script_class_get_name(scr->get_path());
 					String icon_path = EditorNode::get_editor_data().script_class_get_icon_path(name);
 					if (name != StringName() && !icon_path.is_empty()) {
-						category->icon = ResourceLoader::load(icon_path, "Texture");
-						break;
+						Ref<Image> img;
+						img.instantiate();
+						Error err = ImageLoader::load_image(icon_path, img);
+						if (err == OK) {
+							img->resize(16 * EDSCALE, 16 * EDSCALE, Image::INTERPOLATE_LANCZOS);
```

Comment:
The fix looks good, my only doubt is whether it's fine to hardcode 16 here, or if that's something we can read from the editor theme?

Reply:
I'm not sure, I just copy it from `editor_node.h`:
```
Ref<ImageTexture> EditorNode::_load_custom_class_icon(const String &p_path) const {
	if (p_path.length()) {
		Ref<Image> img = memnew(Image);
		Error err = ImageLoader::load_image(p_path, img);
		if (err == OK) {
			img->resize(16 * EDSCALE, 16 * EDSCALE, Image::INTERPOLATE_LANCZOS);
			return ImageTexture::create_from_image(img);
		}
	}
	return nullptr;
}
```
`EDSCALE` is a macro that get editor scale, I think the hard core 16 is a base size, and the real icon size can be auto scale by editor setting.